### PR TITLE
Add the ability to specify an OU upon Domain Join

### PR DESCRIPTION
**Pull request by PrahlM93**

### CODE REVIEW

File: Tests/xComputermanagement.Tests.ps1 [hunk 1]
```diff
@@ -91,7 +91,7 @@ InModuleScope MSFT_xComputer {
             It 'Should return a hashtable containing Name,DomainName, Credential, UnjoinCredential and WorkGroupName' {
```
Comment (tysonjhayes) [inline Tests/xComputermanagement.Tests.ps1]: Should probably add the new keys to this statement.

File: DSCResources/MSFT_xComputer/MSFT_xComputer.psm1 [hunk 2]
```diff
@@ -247,5 +265,18 @@ function GetComputerDomain
     }
 }
 
-Export-ModuleMember -Function *-TargetResource
+function GetComputerOU
```
Comment (tysonjhayes) [inline DSCResources/MSFT_xComputer/MSFT_xComputer.psm1]: Could we name this Get-ComputerOU so we have a proper verb-noun?

**Comment by PrahlM93** [comment]
@tysonjhayes Any other suggestions?

**Comment by tysonjhayes** [comment]
Nothing that needs to be handled in this PR, I'd like to see the verb-noun changed for everything but that's out of scope for this. :grin: 

Looks good to me though, thanks!

**Comment by tysonjhayes** [comment]
@TravisEz13 @KarolKaczmarek either one of you able to lend a hand by merging this so I can close out #6 ?

**Comment by PrahlM93** [comment]
@tysonjhayes, is there any news on whether or not this will be merged?

**Comment by MCrank** [comment]
@tysonjhayes I too would like for this to get merged.  Surprised this was not included initially.

**Comment by tysonjhayes** [comment]
Sorry about the delay here guys. This is now merged in.

**Pull request merged**